scripts/distributed_scripts/run_distributed_xgb.sh

```diff
@@ -0,0 +1,18 @@
+set -e
+
+cd ..
+
+echo "Test distributed mode with XGB..."
+
+### server owns global test data
```

Comment:
Is it support the situation that server owns global test data?

Reply:
Yes, but it is not used since each client owns part of test data, and all clients together can perform the test procedure.

Reply:
So shall we remove the ''redundant'' code here?

Reply:
Modified.